tests/SideBySide.New/ConnectionPool.cs

```diff
@@ -80,37 +80,7 @@ public void ResetConnection(object connectionReset, int poolSize, long expected)
 		}
 
 		[Fact]
-		public async Task ExhaustConnectionPoolWithTimeout()
```

Comment:
MySql.Data uses this value to time out the connection when the pool is full. (This test passes when compiled with mysql-connector-net.) This behaviour makes sense to me and I think we should clone it. (The alternatives would be failing immediately or blocking forever, and none of those seems preferable.)